src/main/java/com/redhat/devtools/intellij/tektoncd/tree/ParentableNode.java

```diff
@@ -32,4 +32,16 @@ public T getParent() {
     public String getName() {
         return name;
     }
+
```

Comment:
This is formally incorrect as it introduce a circular dependency between the 2 classes but we can keep it